indico/htdocs/js/indico/modules/sessions.js

```diff
@@ -0,0 +1,94 @@
+(function(global) {
+    'use strict';
+
+    function setupTableSorter() {
+        $('.sessions .tablesorter').tablesorter({
+            cssAsc: 'header-sort-asc',
+            cssDesc: 'header-sort-desc',
+            cssInfoBlock: 'avoid-sort',
+            cssChildRow: 'session-blocks-row',
+            headerTemplate: '',
+            sortList: [[1, 0]]
+        });
+    }
+
+    function setupPalettePickers() {
+        $('.palette-picker-trigger').each(function() {
+            var $this = $(this);
+            $this.palettepicker({
+                availableColors: $this.data('colors'),
+                onSelect: function(background, text) {
+                    $.ajax({
+                        url: $(this).data('href'),
+                        method: $(this).data('method'),
+                        data: JSON.stringify({'colors': {'text': text, 'background': background}}),
+                        dataType: 'json',
+                        contentType: 'application/json',
+                        error: handleAjaxError,
+                        complete: IndicoUI.Dialogs.Util.progress()
+                    });
+                }
+            });
+        });
+    }
+
+    global.setupSessionsList = function setupSessionsList() {
+        enableIfChecked('.sessions-wrapper', '.select-row', '.js-requires-selected-row');
+        setupTableSorter();
+        setupPalettePickers();
+
+        $('.sessions .toolbar').on('click', '.disabled', function(evt) {
+            evt.preventDefault();
+            evt.stopPropagation();
+        });
+
+        function updateSessionsListOnSuccess(data) {
+            if (data) {
+                $('.sessions-wrapper').html(data.html);
+                setupTableSorter();
+                setupPalettePickers();
+            }
+        }
+
+        $('.sessions').on('indico:htmlUpdated', function() {
+            setupTableSorter();
+            setupPalettePickers();
+        });
+
+        $('.sessions').on('click', '.show-session-blocks', function() {
+            $(this).closest('tr').toggleClass('selected').nextUntil('tr:not(.session-blocks-row)', 'tr').toggle();
+        });
+
+        $('.sessions').on('indico:confirmed', '#remove-selected-sessions', function(evt) {
```

Comment:
You can do this fully declarative like @OmeGak  did in the contribution list:

            <button class="i-button js-enable-if-checked" disabled
                    data-method="POST"
                    data-href="{{ url_for('.manage_delete_contribs', event) }}"
                    data-params-selector="#contribution-list input[name=contribution_id]:checked"
                    data-confirm="{% trans %}Do you really want to delete the selected contributions?{% endtrans %}"
                    data-title="{% trans %}Confirm deletion{% endtrans %}"
                    data-update="#contribution-list">
                {% trans %}Remove{% endtrans %}
            </button>

(obviously needs to be adapted)